README.md

```diff
@@ -1,2 +1,14 @@
 # VideoReviewConsoleApp
 Create Video Reviews in Content Moderator
+## Getting Started
+### For a Single Video File
+Build and run the code, a console should be present with the following messege:
+#### Enter the fully qualified local path for Uploading the video :
+Format of the local path can be in quotes i.e. "C:\{path}" or without quotes i.e. C:\{path}
+### Transcription Option
+Choose y for transcription n for no transcription
```

Comment:
Choosing Y will use your Azure Media Services account to generate the video transcript